app/presenters/content_item/contents_list.rb

```diff
@@ -114,7 +114,9 @@ def no_first_item?
 
     def exclude_contents_list_for_manual_section?
       # MOJ require contents lists for manual section pages.
-      moj_content = content_item["links"]["organisations"].first["content_id"] == "dcc907d6-433c-42df-9ffb-d9c68be5dc4d"
+
+      # moj_content = content_item["links"]["organisations"]&.try(:content_id)&.first == "dcc907d6-433c-42df-9ffb-d9c68be5dc4d"
+      moj_content = content_item["links"]["organisations"] && content_item["links"]["organisations"].first["content_id"] == "dcc907d6-433c-42df-9ffb-d9c68be5dc4d"
```

Comment:
I'm assuming we hit some content items that don't have organisations links? In which case, nice catch!

Reply:
Thanks Pete. I had various test failures related to the previous changes and so have amended like so (which seems to be okay) >

https://github.com/alphagov/government-frontend/blob/7f53b82f9884fdc45a8c51a7523ca93f961fb3e6/app/presenters/content_item/contents_list.rb#L118